.gitignore

```diff
@@ -55,59 +56,49 @@ chains
 
 # tox/pytest cache
 .cache
+.pytest_cache
 
 # Test output logs
 logs
-### JetBrains template
-# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio and Webstorm
-# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
-
-# User-specific stuff:
-.idea/workspace.xml
-.idea/tasks.xml
-.idea/dictionaries
-.idea/vcs.xml
-.idea/jsLibraryMappings.xml
-
-# Sensitive or high-churn files:
-.idea/dataSources.ids
-.idea/dataSources.xml
-.idea/dataSources.local.xml
-.idea/sqlDataSources.xml
-.idea/dynamic.xml
-.idea/uiDesigner.xml
-
-# Gradle:
-.idea/gradle.xml
-.idea/libraries
-
-# Mongo Explorer plugin:
-.idea/mongoSettings.xml
 
 # VIM temp files
 *.sw[op]
 
 # mypy
 .mypy_cache
 
+# macOS
+.DS_Store
+
+# pyenv
+.python-version
+
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio and Webstorm
+# For a more precise, explicit template, see:
+# https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
+
+## General
+.idea/*
+.idea_modules/*
+
 ## File-based project format:
 *.iws
 
-## Plugin-specific files:
+## IntelliJ
+out/
 
-# IntelliJ
-/out/
+## Plugin-specific files:
 
-# mpeltonen/sbt-idea plugin
+### mpeltonen/sbt-idea plugin
 .idea_modules/
```

Comment:
nit: this is covered on line 82 I believe

Reply:
removed, thanks!